ui/tests/acceptance/settings/mount-secret-backend-test.js

```diff
@@ -136,7 +136,7 @@ module('Acceptance | settings/mount-secret-backend', function (hooks) {
     await settled();
     await page.enableEngine();
     await mountBackend('kv', path);
-
+    await settled();
```

Comment:
Has this test been failing because the tests tried to assert the DOM element before it exists? I find that using `waitFor(element)` to be a little more resilient when stabilizing flaky tests.